it/stubs/WireMockStubs.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package stubs
+
+import com.github.tomakehurst.wiremock.client.MappingBuilder
+import com.github.tomakehurst.wiremock.client.WireMock._
+import com.github.tomakehurst.wiremock.http.HttpHeader
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import play.api.http.ContentTypes.JSON
+import play.api.test.Helpers.CONTENT_TYPE
+import uk.gov.hmrc.http.HttpResponse
+
+trait WireMockStubs {
```

Comment:
we have this file repeated again?

Reply:
This was done locally before the changes for the other API were merged to main. Removed